Tests/Opc.Ua.Client.Tests/Opc.Ua.Client.Tests.csproj

```diff
@@ -20,6 +20,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="OpenTelemetry" Version="1.5.1" />
```

Comment:
We should only use .net diagnostics API, no OTEL dependencies.  Where do we need these and is there a .net version of this?

Reply:
removed accidental commit.